src/components/pdfviewer/PdfViewer.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from "react";
+import { pdfBase64Example } from "./pdfbase64example";
+
+const PdfViewer = ({
+  filePdf = pdfBase64Example,
+  width = "100%",
+  height = "700px",
+}) => {
+  const [createdLinks, setCreatedLinks] = useState("");
```

Comment:
Why is it called createdLinks instead of createdLink and the value is a string instead of an array? The name implies that you have an array of strings instead of one single string